tests/unit/parsing/test_sandbox_parse_flow.py

```
@@ -180,39 +179,29 @@ def parsing_service(fake_sandbox):
+    """Stand-in for CodeGraphService call assertions.
 
+    Neo4j is permanently bypassed in ParsingService, so CodeGraphService
+    is no longer imported there. Keep a MagicMock so existing tests can
+    still assert the graph write path was not invoked.
+    """
     instance = MagicMock(name="CodeGraphService_instance")
     instance.store_graph_from_artifacts = MagicMock()
     instance.close = MagicMock()
-    with patch.object(ps_mod, "CodeGraphService", return_value=instance):
-        yield instance
+    yield instance
```

Ruff (0.16.0)

[warning] No teardown in fixture `stub_code_graph_service`, use `return` instead of `yield`

Replace `yield` with `return`

(PT022)